src/client/cockpit-client

```diff
@@ -276,7 +277,8 @@ class ExternalCockpitWs:
 class InternalCockpitWs:
     def start(self):
         self.socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        self.socket.bind(('127.0.90.90', 0))
+        # Use static port to convince WebKit it is the same domain
```

Comment:
A bit more explicit like "static port is needed for WebKit local storage to persist across runs" or so?

Reply:
Done